src/test/java/io/confluent/idesidecar/websocket/resources/WebsocketEndpointTest.java

```diff
@@ -310,15 +321,15 @@ public String processIdString() {
   @Test
   public void testWebsocketLifecycle()
   {
-    var firstWorkspace = connectWorkspace(true);
+    var firstWorkspace = connectWorkspace(true, false);
```

Comment:
connectWorkspace() now has second boolean param for indicating if it should itself consume the first WORKSPACE_COUNT_CHANGED message, useful for a lot of the higher level tests which had to spell that out explicitly.